### ISSUE TITLE
Input Shape Mismatch

### ISSUE THREAD

**Issue by AechPro** [yhenon/keras-frcnn#62]
Hello, I have stated in a previous issue that I am working to re-implement this project in python 3.6. I am using a Theano backend, and I seem to be running into an error when I begin the training process. 
The error is `Exception: Input dimension mis-match. Exception: Input dimension mis-match. (input[0].shape[2] = 45, input[2].shape[2] = 179)`. 
This error is thrown inside the RPN loss function `def rpn_loss_cls_fixed_num(y_true, y_pred)` 
because of `K.binary_crossentropy(y_pred[:, :, :, :], y_true[:, num_anchors:, :, :])`. 
This happens because the output of the RPN classifier is of the shape `(1, 9, 179, 149)`
while the output `y_rpn_cls` from `data_augment.get_anchor_gt` is of the shape `(1, 18, 45, 38)`. 
I'm not sure why these shapes don't match. Any help you can provide would be greatly appreciated.

Let me know if there is any code you need to see to make the process easier and I will provide it as soon as I am able to.

Thanks!

**Comment by AechPro** [comment]
I found the fix. My issue was in not building the base ResNet50 architecture correctly, resulting in a misshapen output.
